data/test_domains/test_domain_from_directory1/domain_invalid.yml

```diff
@@ -0,0 +1,4 @@
+# invalid yml file should create warning
+# contains duplicate key
+version: "2.0"
+version: "2.0"
```

Comment:
Please add a new line here 😄

Reply:
Done. Also I used flake8 instead of black (you might consider to change that in the todo box on top) for linting, so I get the same error messages as your pipeline :-/

Reply:
using flake8 **and** black was a bit of a surprise, however, see my comment below, I get different results on my Mac than your pipeline produces.

Reply:
I actually run `make formatter` before committing and that solves some of the code quality problems. `make lint` is also useful for surfacing any linting issues.

> it suggests to change 79 other files on my pc.
Not sure why locally you run into this issue - might be because you didn't merge `main` into your branch locally?
I will consult with some of my colleagues and get back to you as soon as I learn more.

Reply:
> What is your pipeline using?

Yes, you were right, you'll have to downgrade to black version `19.10b0` which is what we currently use at the moment 😅

Reply:
ok done.
It reverted all the changes made by black 21 😄

Reply:
ok now flake8 and black checks are good, but `main` passed me again, so I needed to merge that again.